src/ice/candidate.rs

```diff
@@ -210,7 +234,13 @@ impl Candidate {
     /// * `addr` - The TURN server's allocated address that will be used for relaying traffic.
     ///            This is the address that will be used for communication with the peer.
     /// * `proto` - The transport protocol to use (UDP, TCP, etc.).
-    pub fn relayed(addr: SocketAddr, proto: impl TryInto<Protocol>) -> Result<Self, IceError> {
+    /// * `local` - The local interface address that corresponds to this candidate. This is the
+    ///             address from which the TURN allocation request was sent.
+    pub fn relayed(
+        addr: SocketAddr,
+        proto: impl TryInto<Protocol>,
+        local: SocketAddr,
```

Comment:
I just updated to latest `main` and in the previous version, these two parameters were flipped. Do we want to flip them again? Then the upgrade would be a tad easier.

Reply:
I don't think it is a breaking change. The return value from `base` is breaking but behaviour of this parameter isn't.